unit_tests/test_lib_charms_ovn_charm.py

```diff
@@ -548,8 +550,10 @@ def test_configure_bridges(self):
         ovsdb.open_vswitch.set.assert_has_calls([
             mock.call('.', 'external_ids:ovn-bridge-mappings',
                       'other:br-data,provider:br-ex'),
-            mock.call('.', 'external_ids:ovn-cms-options',
-                      'enable-chassis-as-gw'),
+        ], any_order=True)
```

Comment:
is it needed? there is only one call in the list. Is it about allowing extra calls? I find the python doc about this not crystal-clear.

Reply:
After removing the extra call it is no longer needed, so you caught me being lazy!